### ISSUE TITLE
Install error with click v8.0.0

### ISSUE THREAD

**Issue by rhysd** [Vimjas/covimerage#95]
click v8.0.0 was released at May 12

https://pypi.org/project/click/#history

And some breaking change due to the major release broke this package.

Example error:

```
Run pip install covimerage
Collecting covimerage
  Downloading covimerage-0.2.1-py3-none-any.whl (16 kB)
Collecting attrs>=16.1.0
  Downloading attrs-21.2.0-py2.py3-none-any.whl (53 kB)
Collecting click
  Downloading click-8.0.0-py3-none-any.whl (96 kB)
Collecting coverage<5.0a6
  Downloading coverage-4.5.4.tar.gz (385 kB)
Using legacy 'setup.py install' for coverage, since package 'wheel' is not installed.
Installing collected packages: coverage, click, attrs, covimerage
    Running setup.py install for coverage: started
    Running setup.py install for coverage: finished with status 'done'
Successfully installed attrs-21.2.0 click-8.0.0 coverage-4.5.4 covimerage-0.2.1
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.9.5/x64/bin/covimerage", line 5, in <module>
    from covimerage.cli import main
  File "/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/covimerage/__init__.py", line 8, in <module>
    from click.utils import string_types
ImportError: cannot import name 'string_types' from 'click.utils' (/opt/hostedtoolcache/Python/3.9.5/x64/lib/python3.9/site-packages/click/utils.py)
Error: Process completed with exit code 1.
```

https://github.com/rhysd/clever-f.vim/runs/2583434786

**Comment by rhysd** [comment]
Note: Workaround for this issue is installing click v7 before installing covimerage.

```sh
pip install 'click<8.0.0'
pip install covimerage
```

**Comment by blueyed** [comment]
Fixed in v0.2.2 (just released).
Sorry that it took so long.
Compatibility with click 8 will be done in https://github.com/Vimjas/covimerage/pull/97.
